hypervisor/arch/x86/vmexit.c

```diff
@@ -311,3 +312,12 @@ static int32_t wbinvd_vmexit_handler(struct acrn_vcpu *vcpu)
 
 	return 0;
 }
+
+/* ACRN doesn't support nested virtualization,
+ * so when guest executes VMX instruction, #UD exception should be injected.
+ */
+static int32_t vmx_op_vmexit_handler(struct acrn_vcpu *vcpu)
```

Comment:
This API name seems to be hard to know.  something link with ud ?  Basically all UD instructions comes here.